composer.json

```diff
@@ -6,8 +6,9 @@
     "require": {
         "php": "^7.3",
         "sylius/sylius": "^1.8",
+        "symfony/messenger": "^4.4 || ^5.2",
         "mollie/mollie-api-php": "^2.0",
-        "sylius/refund-plugin": "1.0.0-RC.5",
+        "sylius/refund-plugin": "1.0.0-RC.8",
         "sylius/admin-order-creation-plugin": "^0.9.0",
```

Comment:
Hello, could you also update this requirement to ^0.11? Sylius updated their plugin to support v1.9 at this tag.
https://github.com/Sylius/AdminOrderCreationPlugin/releases/tag/v0.11.0